.babelrc

```diff
@@ -0,0 +1,4 @@
+{
+  "presets" : ["react", "es2015"],
+  // "plugins": ["transform-es2015-destructuring", "transform-object-rest-spread"]
```

Comment:
So may I ask why you commented out this code?

Reply:
I realised I don't need it for now. Will remove it